lib/internal/modules/esm/resolve.js

```diff
@@ -779,7 +779,7 @@ function packageResolve(specifier, base, conditions) {
 
   // eslint can't handle the above code.
   // eslint-disable-next-line no-unreachable
-  throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base));
+  throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base), null);
```

Comment:
Is there a reason for passing `null` here?
```suggestion
  throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base));
```

Reply:
We need to provide all three arguments to the error, per the error call validation.

Reply:
Ah right, the prototype setter bug.

Reply:
CJS has its own module not found error, and the ESM resolver doesn't delegate to it.

Reply:
Ah, missed that. Apparently they use the same name even though they are different instances. That seems...not ideal. 😅

Reply:
They don't use the same name, although it's close: CJS error is `MODULE_NOT_FOUND`, while here we're touching `ERR_MODULE_NOT_FOUND`.